doc/README-mosquitto.md

```diff
@@ -0,0 +1,151 @@
+# Ecplipse mosquitto_password cracking
+
+mosquitto2john.py is a helper script to allow you to convert Eclipse Mosquitto
+passwd files into a John friendly format. The script takes a 
+[mosquitto_passwd file](https://mosquitto.org/man/mosquitto_passwd-1.html) as 
+its only required input. 
+
+## Usage
+
+```
+❯ mosquitto2john.py --help
+usage: mosquitto2john.py [-h] [-hc] [-o OUT_FILE] passwd_file
+
+positional arguments:
+  passwd_file           Path to the target mosquitto_passwd file.
```

Comment:
Perhaps "source" rather than "target"? (Should also fix this in the script.)

Also, ideally the script would be able to handle multiple files or at least would give an error (or does it already?) if multiple files are listed on the command-line.